alembic/versions/2f28d2536c90_parametrize_over_cluster_memory.py

```diff
@@ -0,0 +1,37 @@
+"""Parametrize over cluster memory
+
+Revision ID: 2f28d2536c90
+Revises: 2764a4f5582b
+Create Date: 2022-09-16 13:43:20.364162
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2f28d2536c90'
+down_revision = '2764a4f5582b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(
+    """update test_run
+    set name = name || '[100% cluster memory]'
```

Comment:
@ian-r-rose I'm not entirely sure what `name` and `orignalname` is. is it correct for me to just modify `name`?

Reply:
`originalname` is the name of the actual function, whereas `name` is the full name with any parametrizations tacked on (this is pytest terminology). So the structure of this migration looks correct to me.